### ISSUE TITLE
[sklearn#34180] `ExtraTree*.fit` docstring is the `DecisionTree*.fit` docstring

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#345]
**Mirrored from [scikit-learn#34180](https://github.com/scikit-learn/scikit-learn/issues/34180)**

Original title: `ExtraTree*.fit` docstring is the `DecisionTree*.fit` docstring
Original labels: Documentation, Needs Triage
Created: 2026-06-02T10:46:39Z

---

> [!WARNING]
> This issue is not yet ready for a PR. If you are interested in contributing to scikit-learn, please have a look at our [contributing guidelines](https://scikit-learn.org/dev/developers/contributing.html), and in particular the sections for [new contributors](https://scikit-learn.org/dev/developers/contributing.html#new-contributors) and the ["Needs triage"](https://scikit-learn.org/dev/developers/contributing.html#issues-tagged-needs-triage) label.

### Describe the issue linked to the documentation

The docstring for `ExtraTree*.fit` is the same than the one of `DecisionTree*` so it's inaccurate:

https://scikit-learn.org/stable/modules/generated/sklearn.tree.ExtraTreeRegressor.html#sklearn.tree.ExtraTreeRegressor.fit

> Build a decision tree regressor from the training set (X, y).

> Return:
>     self : DecisionTreeRegressor
>        Fitted estimator.

### Suggest a potential alternative/fix

Removing `ExtraTree*` classes would fix this (this is one option considered in https://github.com/scikit-learn/scikit-learn/issues/33158)

Otherwise we need to add a fit method to those classes.

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Addressed in upstream PR(s). See scikit-learn/scikit-learn PRs for #34180.
